docs/interpreter/shell.md

```diff
@@ -2,19 +2,39 @@
 layout: page
 title: "Shell Interpreter"
 description: "Shell Interpreter"
-group: manual
+group: interpreter
 ---
 {% include JB/setup %}
 
-## Shell interpreter for Apache Zeppelin
+# Shell interpreter for Apache Zeppelin
 
-### Overview
-Shell interpreter uses [Apache Commons Exec](https://commons.apache.org/proper/commons-exec) to execute external processes. 
+<div id="toc"></div>
 
+## Overview
+Shell interpreter uses [Apache Commons Exec](https://commons.apache.org/proper/commons-exec) to execute external processes. 
 In Zeppelin notebook, you can use ` %sh ` in the beginning of a paragraph to invoke system shell and run commands.
-Note: Currently each command runs as Zeppelin user.
 
-### Example
+> **Note :** Currently each command runs as Zeppelin user.

```

Comment:
perhaps it could help to explain "Zeppelin user" as "the user Zeppelin server is running as" (or similar)?